packages/migrate/src/transforms/2345-fix-transform.ts

```diff
@@ -0,0 +1,28 @@
+import ts from 'typescript';
+
+import FixTransform, { FixedFile } from '../interfaces/fix-transform';
```

Comment:
import FixTransform, { type FixedFile } from '../interfaces/fix-transform';

Reply:
FixedFile itself is an interface. Do I need to say 'type FixedFile' while importing it?